library/templates/dpr_parksproperties.yml

```diff
@@ -0,0 +1,33 @@
+dataset:
+  name: &name dpr_parksproperties
+  version: "{{ version }}"
+  acl: public-read
+  source:
```

Comment:
See comment about opendata sources on the `dcla_culturalinstitutions.yml` file.

Again `format: geojson` in this case.